watertap/examples/flowsheets/case_studies/full_water_resource_recovery_facility/BSM2.py

```diff
@@ -382,6 +378,10 @@ def set_operating_conditions(m):
     m.fs.TU.hydraulic_retention_time.fix(86400 * pyo.units.s)
     m.fs.TU.diameter.fix(10 * pyo.units.m)
 
+    # TODO: resolve the danger of redundant constraint related to pressure equality constraints created in mixer, specifically for isobaric conditions. the mixer initializer will turn these constraints back on
+    for mx in m.mixers:
+        mx.pressure_equality_constraints[0.0, 2].deactivate()
```

Comment:
Does this need to be done here and in the initialization?

Reply:
@MarcusHolly Yes, because the initializer for the mixer apparently reactivates the previously deactivated pressure equality constraints. So right now, we deactivate during the build/setting specifications and also after initializing.